Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```
@@ -0,0 +1,1171 @@
+  type: 0
+description: Use Key Vault to safeguard and manage cryptographic keys and secrets
+  used by cloud applications and services.
+display: Azure Key Vault
+name: AzureKeyVault
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Key Vault name.
+      isArray: false
+      name: vault_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Key Vault supported Azure location. The location cannot be changed
+        after the Key Vault is created. Default value is 'westus'.
+      isArray: false
+      name: location
+      predefined:
+      - eastus
+      - eastus2
+      - southcentralus
+      - westus2
+      - australiaeast
+      - southeastasia
+      - northeurope
+      - uksouth
+      - westeurope
+      - centralus
+      - northcentralus
+      - westus
+      - southafricanorth
+      - centralindia
+      - eastasia
+      - japaneast
+      - jioindiawest
+      - koreacentral
+      - canadacentral
+      - francecentral
+      - germanywestcentral
+      - norwayeast
+      - switzerlandnorth
+      - uaenorth
+      - brazilsouth
+      - centralusstage
+      - eastusstage
+      - eastus2stage
+      - northcentralusstage
+      - southcentralusstage
+      - westusstage
+      - westus2stage
+      - asia
+      - asiapacific
+      - australia
+      - brazil
+      - canada
+      - europe
+      - global
+      - india
+      - japan
+      - uk
+      - unitedstates
+      - eastasiastage
+      - southeastasiastage
+      - centraluseuap
+      - eastus2euap
+      - westcentralus
+      - westus3
+      - southafricawest
+      - australiacentral
+      - australiacentral2
+      - australiasoutheast
+      - japanwest
+      - koreasouth
+      - southindia
+      - westindia
+      - canadaeast
+      - francesouth
+      - germanynorth
+      - norwaywest
+      - switzerlandwest
+      - ukwest
+      - uaecentral
+      - brazilsoutheast
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specify whether the key vault is a standard vault or a premium
+        vault. Default value is 'standard'.
+      isArray: false
+      name: sku_name
+      predefined:
+      - standard
+      - premium
+      required: false
+      secret: false
+    - default: false
+      description: 'The object ID of a user, service principal or security group in
+        the Azure Active Directory tenant for the vault. The object ID must be unique
+        for the list of access policies: Any change in the access policy regards that
+        object ID, will override the exists one. '
+      isArray: false
+      name: object_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Permissions to keys. If the Key Vault exists, you must supply
+        the previous keys permissions inorder to keep them the same. Access policy
```

Again, let's re-review the grammar and phrasinig
```suggestion
        the previous keys' permissions in order to keep them unchanged. Access policy
```

---

Done.